feature-libs/quote/components/details/comment/quote-details-comment.component.html

```diff
@@ -1,14 +1,12 @@
 <ng-container *ngIf="quoteDetails$ | async as quoteDetails">
   <ng-container *ngIf="showComments(quoteDetails)">
-    <div class="cx-comment-toggle" (click)="expandComments = !expandComments">
+    <div class="cx-toggle" (click)="expandComments = !expandComments">
       <cx-icon
         aria-hidden="false"
         [type]="expandComments ? iconTypes.CARET_UP : iconTypes.CARET_DOWN"
       >
       </cx-icon>
-      <span class="cx-comment-text">{{
-        'quote.comments.title' | cxTranslate
-      }}</span>
+      <span class="cx-text">{{ 'quote.comments.title' | cxTranslate }}</span>
```

Comment:
```suggestion
      <span class="cx-toggle-text">{{ 'quote.comments.title' | cxTranslate }}</span>
```
i think cx-text is to commomn, what about cx-toggle-text?

Reply:
I don't agree because `cx-text` is framed by `div` with `cx-toggle` class. 
Please, see  `_quote-toggle.scss` mixin.
The main SPA guideline is to keep it simple and consider within one component.
It is unnecessary to concatenate the classes. In this case the class name will be unnecessary long but we don't win anything through this concatenation only complexity.